sphinx-doc/features.rst

```diff
@@ -27,21 +27,22 @@ At runtime, `hoomd.version.hpmc_built` indicates whether the build supports HPMC
 Molecular dynamics
 ------------------
 
-HOOMD-blue can perform molecular dynamics simulations (`md`) with NVE, NVT, NPT, NPH, Langevin,
-Brownian, overdamped viscous integration methods (`md.methods`), and energy minimization
-(`md.minimize`). Unless otherwise stated in the documentation, all integration methods integrate
-both translational and rotational degrees of freedom. Some integration methods support manifold
-constraints (`md.methods.rattle`). HOOMD-blue provides a number of pair potentials (`md.pair`)
-including pair potentials that depend on particle orientation (`md.pair.aniso`) and many body
-potentials (`md.many_body`). HOOMD-blue also provides bond potentials and distance constraints
-commonly used in atomistic and coarse-grained force fields (`md.angle`, `md.bond`,
-`md.constrain.Distance`, `md.dihedral`, `md.improper`, `md.special_pair`) and can model rigid bodies
-(`md.constrain.Rigid`). External fields `md.external.field` apply potentials based only on the
-particle's position and orientation, including walls (`md.external.wall`) to confine particles in a
-specific region of space. `md.long_range` provides long ranged interactions, including the PPPM
-method for electrostatics. HOOMD-blue enables active matter simulations with `md.force.Active` and
-`md.update.ActiveRotationalDiffusion`. At runtime, `hoomd.version.md_built` indicates whether the
-build supports MD simulations.
+HOOMD-blue can perform molecular dynamics simulations (`md`) with constant volume, constant
+pressure, Langevin, Brownian, overdamped viscous integration methods (`md.methods`), and energy
+minimization (`md.minimize`). The constant volume and constant pressure methods may be applied with
+or without a thermostat (`md.methods.thermostats`). Unless otherwise stated in the documentation,
+all integration methods integrate both translational and rotational degrees of freedom. Some
+integration methods support manifold constraints (`md.methods.rattle`). HOOMD-blue provides a number
+of pair potentials (`md.pair`) including pair potentials that depend on particle orientation
+(`md.pair.aniso`) and many body potentials (`md.many_body`). HOOMD-blue also provides bond
```

Comment:
This implies that many body potentials are pair potentials, the sentence needs to be split up.

Reply:
Isn't this just a case of missing commas?

Reply:
I reworded this for clarity, including the Oxford comma.